slicing_indexing.py

```diff
@@ -0,0 +1,16 @@
+#Create a python program to do specified string indexing operations & github requirements
```

Comment:
Middle characters printed are wrong for an even length string.

<img width="456" alt="image" src="https://user-images.githubusercontent.com/38807024/203932854-67a59188-0264-48d0-bf08-4ed1bef472ff.png">

Middle character is not printed.

<img width="438" alt="image" src="https://user-images.githubusercontent.com/38807024/203933052-4e529012-740d-43c8-95f7-786c36542552.png">